.gitlab-ci.yml

```diff
@@ -0,0 +1,55 @@
+image: golang:1.11.4
+
+variables:
+  REPO_NAME: github.com/mendersoftware/mender
+
+before_script:
+  - mkdir -p /go/src/$(dirname $REPO_NAME)/mender /go/src/_/builds
+  - cp -r $CI_PROJECT_DIR /go/src/$(dirname $REPO_NAME)
+  - cd $GOPATH/src/$REPO_NAME
+  - apt-get update && apt-get install -yyq liblzma-dev
+  - make get-tools
+
+stages:
+  - test
+  - build
+  - publish
+
+test:
+  stage: test
+  script:
+    - make extracheck
+    - make coverage
+    - mkdir -p tests/unit-coverage && find . -name 'coverage.txt' -exec cp --parents {} ./tests/unit-coverage \;
+    - tar -cvf $CI_PROJECT_DIR/unit-coverage.tar tests/unit-coverage
+  tags:
+    - mender-qa-slave
```

Comment:
We need to remove this tag or share the runner with the project. I vote for just removing the tag, as I would like all the open Pipelines to run in public GitLab runners

Reply:
See job here: https://gitlab.com/Northern.tech/Mender/mender/-/jobs/240224541

Reply:
Unfortunately the checks seem to be a bit too much for the public runners, since they failed most of the time. So for now I enabled the shared runner or should we maybe add an additional like-public runner of our own?